R/sim.cs-utils.r

```diff
@@ -32,6 +32,10 @@ ldpar <- function(
   return(spar);
 }
 
+#' @rdname ldpar
+#' @export
+row_longitudinal_parameter <- ldpar
```

Comment:
- [ ] do the `lifecycle` process for this function
- [ ] don't export this function